Makefile

```diff
@@ -1,401 +1,95 @@
-VERSION_DATE ?= $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
-PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x
-GOARCH ?= $(go env GOARCH)
-GOOS ?= $(go env GOOS)
-GO_FLAGS ?= GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=0 GO111MODULE=on
-KUBERNETES_CONFIG ?= "$(HOME)/.kube/config"
-WATCH_NAMESPACE ?= ""
-BIN_DIR ?= "build/_output/bin"
-IMPORT_LOG=import.log
-FMT_LOG=fmt.log
 
-OPERATOR_NAME ?= jaeger-operator
-NAMESPACE ?= "$(USER)"
-BUILD_IMAGE ?= "$(NAMESPACE)/$(OPERATOR_NAME):latest"
-IMAGE_TAGS ?= "--tag $(BUILD_IMAGE)"
-OUTPUT_BINARY ?= "$(BIN_DIR)/$(OPERATOR_NAME)"
-VERSION_PKG ?= "github.com/jaegertracing/jaeger-operator/pkg/version"
-JAEGER_VERSION ?= "$(shell grep jaeger= versions.txt | awk -F= '{print $$2}')"
-OPERATOR_VERSION ?= "$(shell git describe --tags)"
-STORAGE_NAMESPACE ?= "${shell kubectl get sa default -o jsonpath='{.metadata.namespace}' || oc project -q}"
-KAFKA_NAMESPACE ?= "kafka"
-KAFKA_EXAMPLE ?= "https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/0.19.0/examples/kafka/kafka-persistent-single.yaml"
-KAFKA_YAML ?= "https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.19.0/strimzi-cluster-operator-0.19.0.yaml"
-ES_OPERATOR_NAMESPACE ?= openshift-logging
-ES_OPERATOR_BRANCH ?= release-4.4
-ES_OPERATOR_IMAGE ?= quay.io/openshift/origin-elasticsearch-operator:4.4
-SDK_VERSION=v0.18.2
-GOPATH ?= "$(HOME)/go"
-GOROOT ?= "$(shell go env GOROOT)"
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
```

Comment:
jaeger-operator? Take a look at the Makefile for the otel-operator, I use another logic to build the image name.

Reply:
This is the default makefile of the bootstraped project, I'll take a look at the otel-operator. thanks !